x/emissions/keeper/keeper.go

```diff
@@ -1155,10 +1155,8 @@ func (k *Keeper) RemoveStake(
 	topicId TopicId,
 	reputer sdk.AccAddress,
 	stake Uint) error {
-	// Run checks to ensure that the stake can be removed, and then update the types all at once, applying rollbacks if necessary
-
 	if stake.IsZero() {
-		return errors.New("stake must be greater than zero")
+		return nil
 	}
```

Comment:
So we gonna keep this `nil`?

Reply:
as we talked ab on slack -- should be an "app layer" decision wrt what to do about throwing or not